central/networkbaseline/service/service_impl.go

```diff
@@ -155,6 +214,25 @@ func (s *serviceImpl) getStatusesForPeers(
 	return statuses
 }
 
+// anonymizedPeerKey anonymizes discovered external peers to the internet
+// for the purposes of looking up matching baseline peers.
+func (s *serviceImpl) anonymizedPeerKey(peer *v1.NetworkBaselineStatusPeer) networkgraph.Entity {
+	entity := peer.GetEntity()
+	if entity.GetType() == storage.NetworkEntityInfo_EXTERNAL_SOURCE && entity.GetDiscovered() {
+		internet := networkgraph.InternetEntity()
+		// removing some fields for map keying
+		return networkgraph.Entity{
+			Type: internet.Type,
+			ID:   internet.ID,
+		}
```

Comment:
```suggestion
		return networkgraph.InternetEntity()
```

Maybe the comment I deleted is more applicable to the other return statement.

Reply:
This was deliberate - we don't want to return the entire internet entity because it contains additional fields that we don't want to key on when looking up in the baselinePeerById map on line 194.